wurst/lib/Lodash.wurst

```diff
@@ -17,71 +17,53 @@ public abstract class Callable
         if cleanup
             destroy this
 
-    function destroyNextTick()
-        nullTimer(-> destroy this)
-
+public abstract class EmptyVoidFunction extends Callable
```

Comment:
Where is this used? You added it but I don't see anything that references it.

Reply:
Hmm I may not have ended up using it. Either way it could be useful in the future.